token/program-2022/src/extension/confidential_transfer/instruction.rs

```diff
@@ -155,20 +153,18 @@ pub enum ConfidentialTransferInstruction {
 
     /// Withdraw SPL Tokens from the available balance of a confidential token account.
     ///
-    /// Fails if the source or destination accounts are frozen.
+    /// Fails if the account is frozen.
     ///
     /// Accounts expected by this instruction:
     ///
     ///   * Single owner/delegate
-    ///   0. `[writable]` The source SPL Token account with confidential transfers configured.
-    ///   1. `[writable]` The destination SPL Token account.
+    ///   1. `[writable]` The SPL Token account.
     ///   2. `[]` The token mint.
     ///   3. `[]` Instructions sysvar.
     ///   4. `[signer]` The single source account owner.
     ///
     ///   * Multisignature owner/delegate
-    ///   0. `[writable]` The source SPL Token account with confidential transfers configured.
-    ///   1. `[writable]` The destination SPL Token account.
+    ///   1. `[writable]` The SPL Token account.
```

Comment:
these should be zero-indexed for consistency with the rest of the instruction descriptions. also i notice the accounts in the comments for `Transfer` and `TransferWithFee` are one-indexed